src/Aspire.Hosting/ContainerResourceBuilderExtensions.cs

```diff
@@ -177,9 +177,9 @@ public static IResourceBuilder<ContainerResource> AddContainer(this IDistributed
     /// <param name="builder">The resource builder.</param>
     /// <param name="args">The arguments to be passed to the container run command when the container resource is started.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
-    public static IResourceBuilder<T> WithContainerRunArgs<T>(this IResourceBuilder<T> builder, params string[] args) where T : ContainerResource
+    public static IResourceBuilder<T> WithContainerHostRunArgs<T>(this IResourceBuilder<T> builder, params string[] args) where T : ContainerResource
```

Comment:
(minor) is there something we can add/change/enhance on the XML docs to help callers understand the difference between this and WithArgs?

Maybe we can add a `<remarks>` section?

Reply:
Moving this to https://github.com/dotnet/aspire/pull/3950/files#r1579935814